src/org/zaproxy/zap/extension/imagelocationscanner/resources/help/toc.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<!DOCTYPE toc
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp TOC Version 2.0//EN"
+         "http://java.sun.com/products/javahelp/toc_2_0.dtd">
+
+<toc version="2.0">
+	<tocitem text="ZAP User Guide" tocid="toplevelitem">
+		<tocitem text="Add Ons" tocid="addons">
+			<tocitem text="Image Location and Privacy Scanner" target="imagelocationscanner"/>
+			<tocitem text="About" target="imagelocationscanner"/>
```

Comment:
- This `tocitem` needs to be child of the above `tocitem`.
- `target` needs to be updated (`imagelocationscanner.about`).